index.html

```diff
@@ -52,4 +52,114 @@ <h1 class="intro height">I’m Kartikay Glad to see you!</h1>
       </ul>
     </div>
   </section>
+  <section>
+    <div class="grid-container">
+      <div class="img-container">
+        <img class="work-image" src="./Images/tonic-one.png" alt="work-Image" />
+      </div>
+      <div class="card">
+        <div class="card-element">
+          <h2 class="card-title">Tonic</h2>
+          <div class="card-section">
+            <p class="para-one">CANOPY</p>
+            <i class="fa fa-circle"></i>
+            <p class="para-two">Back End Dev</p>
+            <i class="fa fa-circle"></i>
+            <p class="para-two">2015</p>
+          </div>
+          <p class="card-body">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="tech-stack">
+            <button class="tech-items" type="button">html</button>
+            <button class="tech-items" type="button">css</button>
+            <button class="tech-items" type="button">javascript</button>
+          </div>
+          <a class="card-button" href="#">See Project</a>
+        </div>
+      </div>
+      <div class="img-container multipost">
+        <img class="work-image" src="./Images/Multiple-one.png" alt="work-Image" />
+      </div>
+      <div class="card multipost">
+        <div class="card-element">
+          <h2 class="card-title">
+            Multi-Post Stories
+          </h2>
+          <div class="card-section">
+            <p class="para-one">CANOPY</p>
+            <i class="fa fa-circle"></i>
+            <p class="para-two">Back End Dev</p>
+            <i class="fa fa-circle"></i>
+            <p class="para-two">2015</p>
+          </div>
+          <p class="card-body">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="tech-stack">
+            <button class="tech-items" type="button">html</button>
+            <button class="tech-items" type="button">css</button>
+            <button class="tech-items" type="button">javascript</button>
```

Comment:
- [ ] This list of technologies should be in a `ul` tag and each technology should be a `li` tag. As this is a requirement for this project, it would be great if you could rectify this issue for all your project cards 👍

Reply:
Thanks for the valuable suggestion. I have implemented all the changes.